cloudant-http/src/main/java/com/cloudant/http/internal/interceptors/IamCookieInterceptor.java

```diff
@@ -52,6 +53,7 @@ public IamCookieInterceptor(String apiKey, String baseUrl) {
             //all JVMs should support UTF-8, so this should not happen
             throw new RuntimeException(e);
         }
+        this.proxyURL = proxyURL;
```

Comment:
Not needed, this is done in the `super`

Reply:
Removed in a1c95fad4ccf0e845fe4649613fc184c33168220